### ISSUE TITLE
Forestry integration

### ISSUE THREAD

**Issue by lawremi** [iguanaman/HungerOverhaul#9]
What is the status of this? Would be happy to contribute. Besides the obvious, been thinking of ways to integrate the breeding mechanim...

**Comment by iguanaman** [comment]
Absolutely.  Unfortunately, this got lost in the 1.5 > 1.6 move and I've not had time to bring it back.  Also, Pam herself mentioned wanting to add Forestry integration to base HC.  Not sure if that is still going ahead.

One of the things that make it tricky to integrate with other mods is that the crops have tile entities which contain the important data.  I've already tried making them compatible with Thaumcraft but it only accepts crops that have the growth stage stored as metadata.
